lib/resque/worker.rb

```diff
@@ -813,17 +813,11 @@ def solaris_worker_pids
     #   RESQUE_PROCLINE_PREFIXresque-VERSION: STRING
     def procline(string)
       $0 = "#{ENV['RESQUE_PROCLINE_PREFIX']}resque-#{Resque::Version}: #{string}"
-      log! $0
+      log :debug, $0
     end
 
-    # Log a message to Resque.logger
-    # can't use alias_method since info/debug are private methods
-    def log(message)
-      info(message)
-    end
-
-    def log!(message)
-      debug(message)
+    def log(severity, message)
```

Comment:
That paranoia is a good thing, and I think you're right. I'm pretty sure this actually is a public signature. I'll adjust it in a bit. Good catch!